enterprise/backend/test/metabase_enterprise/advanced_permissions/models/permissions/block_permissions_test.clj

```diff
@@ -226,7 +226,7 @@
                   (doseq [[message perms] {"with full DB perms"                   (perms/data-perms-path (mt/id))
                                            "with perms for the Table in question" (perms/table-query-path (mt/id :venues))}]
                     (mt/with-temp Permissions [_ {:group_id group-2-id, :object perms}]
-                      (testing "Should be able to run the query"
+                      (testing (format "Should be able to run the query %s" message)
```

Comment:
Here's one example: `message` above was unused but it was clearly meant to be part of the `testing` context